src/Hazelcast.Net/HazelcastClient.Objects.cs

```diff
@@ -189,5 +190,8 @@ Task<IHRingBuffer<T>> GetRingBufferAsync<T>(string name)
             return await task.CfAwait();
 #endif
         }
+
+        /// <inheritdoc />
+        public Task<ISqlService> GetSqlServiceAsync() => Task.FromResult<ISqlService>(_sqlService);
```

Comment:
Why is this getter async? We don't perform any invocations while constructing the SQL?

Also a note, this is called just `Sql` in the Java, not `SqlService`.

Reply:
Added this in case if future versions will require some async operation to initialize the service. Now not sure it's worth it, can do it similar to `HazelcastClient.CPSubsystem` property. @zpqrtbnk what do you think?

Regarding the naming - will update.